packages/amplify-category-function/src/provider-utils/awscloudformation/index.ts

```diff
@@ -141,61 +141,27 @@ async function openEditor(context, category, options: FunctionParameters | Funct
 
 export async function invoke(context, category, service, resourceName) {
   const { amplify } = context;
+  const resourcePath = path.join(amplify.pathManager.getBackendDirPath(), category, resourceName);
   const resourceQuestions = [
-    {
-      type: 'input',
-      name: 'handlerFilename',
-      message: 'Provide the name of the file that contains your handler function:',
-      validate: amplify.inputValidation({
-        operator: 'regex',
-        value: '^[a-zA-Z0-9._-]+$',
-        onErrorMsg: 'You can use the following characters: a-z A-Z 0-9 . - _',
-        required: true,
-      }),
-      default: 'index.js',
-    },
-    {
-      type: 'input',
-      name: 'handlerName',
-      message: 'Provide the name of the handler function to invoke:',
-      validate: amplify.inputValidation({
-        operator: 'regex',
-        value: '^[a-zA-Z0-9_]+$',
-        onErrorMsg: 'You can use the following characters: a-z A-Z 0-9 _',
-        required: true,
-      }),
-      default: 'handler',
-    },
     {
       type: 'input',
       name: 'eventName',
-      message: 'Provide the relative path to the event JSON object:',
+      message: `Provide the path to the event JSON object relative to ${resourcePath}`,
```

Comment:
Why does it have to be a relative path to the resource?

Reply:
It doesn't have to be, but the event is most likely in that folder and this prevents the user from having to type / copy past a long absolute path

Reply:
the message suggests that it will not work with absolute path, that's why I'm pointing it out